### PR TITLE
Create autocomplete component

### PR DESCRIPTION
**Pull request by monkemedia**

### CODE REVIEW

File: components/search/AutoSuggest.vue [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+      <b-list-group
+        v-if="isActive"
+        class="auto-suggest-dropdown"
+      >
+        <b-list-group-item
+          v-for="(value, name, index) in options"
+          :key="index"
+          :href="name"
+          :class="{ 'highlighted': index === focus }"
+          class="auto-suggest-dropdown-list"
```
Comment (mirjamverloop) [inline components/search/AutoSuggest.vue]: This class seems unused.

File: components/search/AutoSuggest.vue [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+      }
+    }
+  };
+</script>
+
+<style lang="scss" scoped>
+  @import "./assets/scss/variables.scss";
+
+  .auto-suggest {
+    position: relative;
```
Comment (mirjamverloop) [inline components/search/AutoSuggest.vue]: This is duplicate, the element has already a style "position: relative".

File: components/search/AutoSuggest.vue [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+    &-dropdown {
+      position: absolute;
+      top: 50px;
+      width: 100%;
+      z-index: 20;
+
+      a.list-group-item {
+        border: 0;
+        box-shadow: none;
```
Comment (mirjamverloop) [inline components/search/AutoSuggest.vue]: Reset the border-radius, as now each list-group-item has rounded corners.

File: components/search/AutoSuggest.vue [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+        padding: .75rem 1.25rem;
+
+        &.highlighted {
+          background-color: #f1f1ee;
```
Comment (mirjamverloop) [inline components/search/AutoSuggest.vue]: There is a variable for this color. Can you use that one?

File: components/search/AutoSuggest.vue [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+  .input-group {
+    width: 100%;
+
+    .input-group-prepend {
+      align-items: center;
+      background-color: $lightgrey;
+      padding-left: .75rem;
+      padding-right: .1rem;
+      border-radius: 0.375rem 0 0 0.375rem;
+    }
```
Comment (mirjamverloop) [inline components/search/AutoSuggest.vue]: Is this still used?

Comment (monkemedia) [reply components/search/AutoSuggest.vue]: It does now as its required for the entity page. I have just stopped the autosuggest functionality when on the entity page

File: components/search/AutoSuggest.vue [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+        >
+      </b-button>
+      <b-list-group
+        v-if="isActive"
+        class="auto-suggest-dropdown"
+      >
+        <b-list-group-item
+          v-for="(value, name, index) in options"
+          :key="index"
+          :href="name"
```
[the remark this answers is not in the document]
Comment (mirjamverloop) [reply components/search/AutoSuggest.vue]: You are right, we can still update that as part of the implementation ticket.

**Pull request merged**